lib/_tls_common.js

```diff
@@ -153,6 +153,38 @@ exports.createSecureContext = function createSecureContext(options) {
     }
   }
 
+  const engineKey = options.engineKey;
+  const privateKeyEngine = options.privateKeyEngine;
```

Comment:
Nit: Can be shortened to

```javascript
const { engineKey, privateKeyEngine } = options;
```

(And probably include `sigalgs` as well if you like.)